.github/pull_request_template.md

```diff
@@ -8,8 +8,23 @@
 
 closes #
 
-## Pull Request Checklist
+## Definition of Done
 
-- [ ] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
-- [ ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style)
-- [ ] If submitting code, please run `mvn clean install -DskipTests` locally before committing
+_Not all items need to be done depending on the issue and the pull request._
+
+* [ ] All commit messages match our [commit message guidelines](https://github.com/zeebe-io/zeebe/blob/develop/CONTRIBUTING.md#commit-message-guidelines)
+
+Code changes:
+* [ ] The submitting code follows our [code style](https://github.com/zeebe-io/zeebe/wiki/Code-Style) (verify by running `mvn clean install -DskipTests` locally)
```

Comment:
I wonder whether we need this and the `all commit messages match ...` checks. They are already checked by our CI pipeline.

Reply:
I'm also fine with it :+1: 

These points were mainly for external contributions. But it is fair to say that this is part of the contribution guide.

Reply:
I was unaware that our CI pipeline is not run for community contributions. I now think we should leave it in.

Reply:
@korthout, thank you for your input. I removed these two points for now because they are part of the contribution guide. 
However, I think that we should improve the guide to make it easier for external contributions.